Unknown timezone "Europe/Amsterdam" with proxmox_virtual_environment_time

**Describe the bug**
I want to set the timezone, but it said it is an unknown timezone

**To Reproduce**
```
resource "proxmox_virtual_environment_time" "node1_timezone" {
 node_name = "node1"
 time_zone = "Europe/Amsterdam"
}
```
**Expected behavior**
I have set the timezone by hand to Europe/Amsterdam
dpkg-reconfigure tzdata
cat /etc/timezone
The timezone is also defined in /usr/share/zoneinfo/zone.tab
In the admin web interface, If I go to "the server / System / Time", I can see the time zone set to "Europe/Amsterdam"

**Screenshots**
![image](https://github.com/bpg/terraform-provider-proxmox/assets/11454484/77c4727f-88a2-440b-a434-49fea2ded275)

**Additional context**
Add any other context about the problem here.
Reading the timezone also gives the same error, which is weird, because it is to read the zone:
```
data "proxmox_virtual_environment_time" "get_time_node1" {
  node_name = "node1"
}
```

- Single or clustered Proxmox: Does not matter
- Proxmox version: 8.2.1
- Provider version (ideally it should be the latest version): 
OpenTofu v1.7.1
on windows_amd64
+ provider registry.opentofu.org/bpg/proxmox v0.61.0

- Debug logs (`TF_LOG=DEBUG terraform apply`):
data.proxmox_virtual_environment_hosts.first_node_host_entries: Read complete after 1s [id=vanzandt_hosts]
2024-07-09T12:50:31.096+0200 [ERROR] vertex "data.proxmox_virtual_environment_time.get_time_vanzandt" error: unknown time zone Europe/Amsterdam
2024-07-09T12:50:31.099+0200 [ERROR] provider.terraform-provider-proxmox_v0.61.0.exe: Response contains error diagnostic: tf_rpc=ReadDataSource @module=sdk.proto diagnostic_detail="" tf_provider_addr=registry.terraform.io/bpg/proxmox @caller=/home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov6/internal/diag/diagnostics.go:58 diagnostic_severity=ERROR diagnostic_summary="unknown time zone Europe/Amsterdam" tf_req_id=93df52ec-a321-9c6e-f1da-42730aa54f8c tf_data_source_type=proxmox_virtual_environment_time tf_proto_version=6.6 timestamp="2024-07-09T12:50:31.095+0200"
2024-07-09T12:50:31.103+0200 [ERROR] vertex "data.proxmox_virtual_environment_time.get_time_vanzandt (expand)" error: unknown time zone Europe/Amsterdam

--- a/fwprovider/test/resource_time_test.go
+++ b/fwprovider/test/resource_time_test.go
@@ -27,10 +27,10 @@ func TestAccResourceTime(t *testing.T) {
 			{
 				Config: te.RenderConfig(`resource "proxmox_virtual_environment_time" "node_time" {
 				  node_name = "{{.NodeName}}"
-				  time_zone = "America/New_York"
+				  time_zone = "Europe/Amsterdam"
 				}`),
 				Check: ResourceAttributes("proxmox_virtual_environment_time.node_time", map[string]string{
-					"time_zone": "America/New_York",
+					"time_zone": "Europe/Amsterdam",
 				}),
 			},
 			{

--- a/proxmoxtf/resource/time.go
+++ b/proxmoxtf/resource/time.go
@@ -10,6 +10,7 @@ import (
 	"context"
 	"fmt"
 	"time"
+	_ "time/tzdata" // Load time zone data, see https://pkg.go.dev/time/tzdata
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"

